install/terraform/modules/gke/variables.tf

```diff
@@ -20,21 +20,23 @@ variable "cluster" {
   type        = map(any)
 
   default = {
-    "location"                = "us-west1-c"
-    "name"                    = "test-cluster"
-    "machineType"             = "e2-standard-4"
-    "initialNodeCount"        = "4"
-    "project"                 = "agones"
-    "network"                 = "default"
-    "subnetwork"              = ""
-    "releaseChannel"          = "UNSPECIFIED"
-    "kubernetesVersion"       = "1.27"
-    "windowsInitialNodeCount" = "0"
-    "windowsMachineType"      = "e2-standard-4"
-    "autoscale"               = false
-    "workloadIdentity"        = false
-    "minNodeCount"            = "1"
-    "maxNodeCount"            = "5"
+    "location"                      = "us-west1-c"
+    "name"                          = "test-cluster"
+    "machineType"                   = "e2-standard-4"
+    "initialNodeCount"              = "4"
+    "project"                       = "agones"
+    "network"                       = "default"
+    "subnetwork"                    = ""
+    "releaseChannel"                = "UNSPECIFIED"
+    "kubernetesVersion"             = "1.27"
+    "windowsInitialNodeCount"       = "0"
+    "windowsMachineType"            = "e2-standard-4"
+    "autoscale"                     = false
+    "workloadIdentity"              = false
+    "minNodeCount"                  = "1"
+    "maxNodeCount"                  = "5"
+    "maintenanceExclusionStartTime" = null
+    "maintenanceExclusionEndTime"   = null
```

Comment:
did you mean to add `deletionProtection` in here as well?

Reply:
We didn't see any issue with standard cluster last time when we recreate test clusters. So I'd prefer not add it here unless we see the issue next time.